tests/datasource_service.tftest.hcl

```
@@ -0,0 +1,114 @@
+mock_provider "opslevel" {
+  alias  = "fake"
+  source = "./mock_datasource"
+}
+
+run "datasource_service_given_alias" {
+  providers = {
+    opslevel = opslevel.fake
+  }
+
+  assert {
+    condition     = data.opslevel_service.mock_service_with_alias.alias == "mock-service-alias"
+    error_message = "alias in opslevel_service mock was not set"
+  }
+
+  assert {
+    condition     = data.opslevel_service.mock_service_with_alias.id == null
+    error_message = "expected null id in opslevel_service mock"
+  }
+
+}
+
+run "datasource_service_given_id" {
+  providers = {
+    opslevel = opslevel.fake
+  }
+
+  assert {
+    condition     = data.opslevel_service.mock_service_with_id.alias == null
+    error_message = "expected null alias in opslevel_service mock"
+  }
+
+  assert {
+    condition     = startswith(data.opslevel_service.mock_service_with_id.id, "Z2lk")
```

```suggestion
    condition     = startswith(data.opslevel_service.mock_service_with_id.id, "Z2lkOi8v")
```

I don't really care about the full contents of ID's in mocks, but this would not pass `opslevel.IsID()` https://github.com/OpsLevel/opslevel-go/blob/c7fe4a152207f6c2c2428953acc0666cd9cb8692/scalar.go#L75

```
$ echo -n Z2lkOi8vmock123 | base64 -d
gid://$%
$ echo -n Z2lk | base64 -d
gid%
$ echo -n gid:// | base64
Z2lkOi8v
```